cli/integration/add_component_version_integration_test.go

```diff
@@ -46,32 +44,96 @@ configurations:
       properties:
         username: %[3]q
         password: %[4]q
-`, host, port, user, password)
-	cfgPath := filepath.Join(t.TempDir(), "ocmconfig.yaml")
-	r.NoError(os.WriteFile(cfgPath, []byte(cfg), os.ModePerm))
-
-	t.Logf("Generated config:\n%s", cfg)
-
-	client := internal.CreateAuthClient(registryAddress, user, password)
-
-	resolver, err := urlresolver.New(
-		urlresolver.WithBaseURL(registryAddress),
-		urlresolver.WithPlainHTTP(true),
-		urlresolver.WithBaseClient(client),
-	)
-	r.NoError(err)
-
-	repo, err := oci.NewRepository(oci.WithResolver(resolver), oci.WithTempDir(t.TempDir()))
-	r.NoError(err)
-
-	t.Run("add component-version with plain OCI registry reference", func(t *testing.T) {
-		r := require.New(t)
-
-		componentName := "ocm.software/test-component"
-		componentVersion := "v1.0.0"
-
-		// Create constructor file
-		constructorContent := fmt.Sprintf(`
+`,
+		},
+		{
+			name: "targeting fallback resolvers",
+			cfg: `
+type: generic.config.ocm.software/v1
+configurations:
+- type: credentials.config.ocm.software
+  consumers:
+  - identity:
+      type: OCIRepository
+      hostname: %[1]q
+      port: %[2]q
+      scheme: http
+    credentials:
+    - type: Credentials/v1
+      properties:
+        username: %[3]q
+        password: %[4]q
+- type: ocm.config.ocm.software
+  resolvers:
+  - repository:
+      type: OCIRepository
+      hostname: %[1]q
+`,
+		},
+		{
+			name: "targeting resolvers",
+			cfg: `
+type: generic.config.ocm.software/v1
```

Comment:
the test here is insufficient as it needs more than one repository / resolver config pair to test anything with resolvers. I think we discussed that you add a new one, but I dont see that anywhere

Reply:
I am adding componentReferences for resolvers in the test